src/win/open_patch.h

```diff
@@ -0,0 +1,9 @@
+/* Authors:
+ * 2021 Tamás Hakkel <hakkelt@gmail.com>
+ */
+
+#ifndef OPEN_PATCH_H
+#define OPEN_PATCH_H
+#include <share.h>
+#define open(pathname, flags, ...) _sopen(pathname, flags|_O_BINARY, _SH_DENYNO __VA_OPT__(,) __VA_ARGS__)
```

Comment:
By default, Windows replaces `\n` character to `\n\r`, even if we write binary data to the file. To avoid this, we need to add  `_O_BINARY` flag.